indexing-service/src/main/java/io/druid/indexing/overlord/http/OverlordResource.java

```diff
@@ -186,13 +198,28 @@ public Response getWorkerConfig()
   @Path("/worker")
   @Consumes(MediaType.APPLICATION_JSON)
   public Response setWorkerConfig(
-      final WorkerBehaviorConfig workerBehaviorConfig
+      final WorkerBehaviorConfig workerBehaviorConfig,
+      @QueryParam("author") String author,
```

Comment:
tried passing the object directly but jetty was throwing errors, 
do you mean passing the json string representation of auditInfo object and reading it via objectmapper ?